src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionInitializer.java

```diff
@@ -42,43 +39,44 @@
     protected static final double INITIAL_OUTLIER_PROBABILITY = 0.01;
     protected static final double INITIAL_MEAN_BIAS = 1.0;
     protected static final double INITIAL_BIAS_VARIANCE = 0.1;   // this is an overestimate, but starting small makes it slow for
-    // mean bias to escape a bad initial guess
+                                                                 // mean bias to escape a bad initial guess
+    public static final AllelicBiasParameters INITIAL_PARAMETERS =
+        new AllelicBiasParameters(INITIAL_MEAN_BIAS, INITIAL_BIAS_VARIANCE, INITIAL_OUTLIER_PROBABILITY);
+
     protected static final double LOG_LIKELIHOOD_CONVERGENCE_THRESHOLD = 0.5;
     protected static final int MAX_ITERATIONS = 50;
 
     //define maxima of search intervals for maximum likelihood -- parameter values above these would be ridiculous
-    private static final double MAX_REASONABLE_OUTLIER_PROBABILITY = 0.1;
-    protected static final double MAX_REASONABLE_MEAN_BIAS = 5.0;
-    protected static final double MAX_REASONABLE_BIAS_VARIANCE = 1.0;
+    public static final double MAX_REASONABLE_OUTLIER_PROBABILITY = 0.1;
+    public static final double MAX_REASONABLE_MEAN_BIAS = 5.0;
+    public static final double MAX_REASONABLE_BIAS_VARIANCE = 1.0;
 
     //the minor allele fraction of a segment must be less than one half by definition
     private static final double MAX_MINOR_ALLELE_FRACTION = 0.5;
 
-    // constants for Brent optimization
-    protected static final MaxEval BRENT_MAX_EVAL = new MaxEval(1000);
-    private static final double RELATIVE_TOLERANCE = 0.001;
-    private static final double ABSOLUTE_TOLERANCE = 0.001;
-    protected static final BrentOptimizer OPTIMIZER = new BrentOptimizer(RELATIVE_TOLERANCE, ABSOLUTE_TOLERANCE);
-
-    private AlleleFractionState state;
+    private AllelicBiasParameters parameters;
+    private AlleleFractionState.MinorFractions minorFractions;
     private static final Logger logger = LogManager.getLogger(AlleleFractionInitializer.class);
 
     /**
      * Do the initialization
      * @param data data
      */
     public AlleleFractionInitializer(final AlleleFractionData data) {
-        state = new AlleleFractionState(INITIAL_MEAN_BIAS, INITIAL_BIAS_VARIANCE, INITIAL_OUTLIER_PROBABILITY, initialMinorFractions(data));
+        parameters = INITIAL_PARAMETERS;
```

Comment:
One disadvantage of breaking up the state is the doubling of lines here and below.